tests/test_selection.py

```diff
@@ -138,3 +138,36 @@ def test_fsel_output_values():
 
 def test_bsel_output_values():
     output_values(backward_output)
+
+
+# -----------------------------------------------------------------------------
+# Testing that forward selection works with 'aic' and 'bic' criterion
+# -----------------------------------------------------------------------------
+
+def test_fsel_aic_output():
+    forward_output = forward(n_features=2, min_change=None, criterion='aic')
+    len(forward_output) == 2
+
+def test_fsel_bic_output():
+    forward_output = forward(n_features=2, min_change=None, criterion='bic')
+    len(forward_output) == 2
+
+# -----------------------------------------------------------------------------
+# Testing that forward selection works with 'min_change' arg
+# -----------------------------------------------------------------------------
+
+def test_fsel_min_change_output():
+    forward_output = forward(n_features=None, min_change=1, criterion=None)
+    len(forward_output) > 1
```

Comment:
@topspinj 

Doesn't this need an `assert` statement (perhaps not for coverage, but to make it a valid test).

Reply:
@TariqAHassan nice catch. fixed